.github/workflows/test.yml

```diff
@@ -39,13 +38,51 @@ jobs:
       - name: Type checks
         run: yarn check:types && yarn flow check
 
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Node
+        uses: ./.github/actions/node
+
+      - name: Build
+        env:
+          GITLAB_TOKEN: ${{ secrets.GITLAB_TOKEN }}
+        run: |
```

Comment:
I do not think we need to build everything for the lint job. I think only tokens here have to be built. Definitely themer, tracking can be excluded. Es and lib folders are ignored, so components also can be excluded from build 🤔

Reply:
I tried it locally and it worked but not on CI. It needs all the packages built 
<img width="793" alt="image" src="https://github.com/kiwicom/orbit/assets/6265045/18b42d9f-5d0a-4bb0-b1a7-eddd7075766d">